### ISSUE TITLE
Missing Information on Loot Tables

### ISSUE THREAD

**Issue by Svarr** [MicrosoftDocs/minecraft-creator#367]
Please add documentation of the `type` property (sibling to `pools`) in the loot table json, along with possible values for it and the `type` property of entries.

The former is used by the vanilla `village_snowy_house.json` loot table (among others, I assume):
```json
{
  "type": "minecraft:chest",
  "pools": [
    //...
  ]
}
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dd485b26-f072-c98a-9e59-e5b12a3ce3ea
* Version Independent ID: 36631caa-b8f2-e4c4-78a1-82618cf95901
* Content: [Introduction to Loot Tables](https://docs.microsoft.com/en-us/minecraft/creator/documents/introductiontoloottables#feedback)
* Content Source: [creator/Documents/IntroductionToLootTables.md](https://github.com/MicrosoftDocs/minecraft-creator/blob/main/creator/Documents/IntroductionToLootTables.md)
* Product: **gaming**
* GitHub Login: @neonerz
* Microsoft Alias: **v-JDHeaden**

**Comment by Svarr** [comment]
Where? Neither the Json Reference (which doesn't contain any documentation on loot tables in the first place) nor the _Loot and Trading Tables_ tutorial pages even mention the `type` property of loot tables.

**Comment by iconicNurdle** [comment]
@Svarr You're right - there is not much about "type" except for examples in the code and a note near the bottom of the "Loot Table Conditions" document about "Exploring the different functions and conditions available can allow creators to really customize the player experience."

My research shows that within the loot table sections of mobs, the main use of "type" is as a field within "entries" within a pool. The only entries for "type" are "item", "loot_table", and "empty."

There is some confusion because a previous release added a content error; before that `"type" : "minecraft:chest"` would have parsed, but looks like it would have caused a crash when something tried to use the loot table.

I hope this helps.

**Comment by Svarr** [comment]
@iconicNurdle I just checked the file I mentioned above and the one included in the bedrock samples for version 1.20.0.1 (the latest I've currently got) still contains the `"type": "minecraft:chest",` line. As it turns out, at least the files `village_fletcher.json`, `village_mason.json`, `village_plains_house.json`, `village_savanna_house.json` and `village_snowy_house.json` in the `\loot_tables\chests\village\` folder contain this field.

Furthermore I found out, that the loot table `village_snowy_house.json` is used by (at least) `vanilla\structures\village\snowy\houses\snowy_small_house_5.nbt`, though I can't tell whether this structure is used as well.
